guacamole/src/main/frontend/src/app/settings/templates/settingsUsers.html

```diff
@@ -43,7 +43,7 @@
                 <td class="username">
                     <a ng-href="#/manage/{{manageableUser.dataSource | escape}}/users/{{manageableUser.user.username | escape}}">
                         <div class="icon user"></div>
-                        <span class="name">{{manageableUser.user.username}}</span>
+                        <span class="name" ng-class="{'user-disabled' : (manageableUser.user.attributes['disabled'] !== null && manageableUser.user.attributes['disabled'] === 'true')}">{{manageableUser.user.username}}</span>
```

Comment:
Same here - the logic behind this test should be encapsulated within the controller.